lib/Runtime/Library/TypedArray.cpp

```diff
@@ -1730,6 +1730,24 @@ namespace Js
         return JavascriptArray::CopyWithinHelper(nullptr, typedArrayBase, typedArrayBase, length, args, scriptContext);
     }
 
+    Var TypedArrayBase::GetKeysEntriesValuesHelper(Arguments& args, ScriptContext *scriptContext, LPCWSTR apiName, JavascriptArrayIteratorKind kind)
```

Comment:
it is for consistency in the file. Look for other usage in the .h file.

---
In reply to: [79927525](https://github.com/Microsoft/ChakraCore/pull/1618#discussion_r79927525) [](ancestors = 79927525)